### ISSUE TITLE
Superslicer 2.5.59.0 Arachne generator gaps in perimeters/walls

### ISSUE THREAD

**Issue by kot0005** [supermerill/SuperSlicer#3394]
### What happened?

i am getting gaps in-between walls/perimeters , more pronounced near angle'd sections

### Project file & How to reproduce

Superslicer 2.5.59.0 Arachne generator :

![20221102_154349](https://user-images.githubusercontent.com/68181325/199416750-b0f05af9-5a40-4e13-8195-6c6aa1708640.jpg)
![20221102_154342](https://user-images.githubusercontent.com/68181325/199416804-3ec0562f-0ff8-450f-ad9c-64921b2212b0.jpg)

This image is using cura 5.2.1 with arachne 

![20221102_154323](https://user-images.githubusercontent.com/68181325/199416951-a8c9acf7-5af8-4d60-a632-41a1a3f32172.jpg)

i tried a few settings like wall thickness, top flow rate etc but nothing seemed to change

### Version

2.5.59.0

### Operating system

windows 10

**Comment by neophyl** [comment]
Yep, PS does this too with some models.  Its why I think the PS implementation isn't ready to be made the default like they have done.

**Comment by diggit** [comment]
It looks like typical pressure advance configuration issue. Too high pressure advance causes such gaps in corners.
Or is it also visible in gcode?

**Comment by CCS86** [comment]
Checking the grams of filament used when slicing with classic vs arachne perimeters generally shows a significant reduction with arachne. This seems to corroborate the under-extrusion people are observing.

**Comment by kot0005** [comment]
> Checking the grams of filament used when slicing with classic vs arachne perimeters generally shows a significant reduction with arachne. This seems to corroborate the under-extrusion people are observing.

oh what, i will turn it off and try

**Comment by kot0005** [comment]
those settings dont work, just using standard super slicer for now for now
